python/main.py

```diff
@@ -22,11 +23,33 @@
 def root():
     return {"message": "Hello, world!"}
 
+# ---> STEP3-2 modify 
 @app.post("/items")
-def add_item(name: str = Form(...)):
+def add_item(name: str = Form(...), category: str = Form(...)):
     logger.info(f"Receive item: {name}")
+    
+    items_list = {"items" : []}
+    if os.path.isfile("items.json") == True:
```

Comment:
```suggestion
    if os.path.isfile("items.json"):
```
isfileが返す変数がすでにbool値なので `== True`は不必要ですね

Reply:
ありがとうございます！